src/engine/BMInterfaceGame.php

```diff
@@ -49,15 +48,29 @@ public function create_game(
         }
 
         try {
-            $gameId = $this->insert_new_game($playerIdArray, $maxWins, $description, $previousGameId);
+            if (!isset($currentPlayerId)) {
+                $currentPlayerId = $playerIdArray[0];
```

Comment:
Ten seconds of thinking: we think that throwing an exception should be fine; no one should be able to create games without a current player right now because it's in the authenticated part of BMInterface.php.